test/e2e/go.mod

```diff
@@ -7,7 +7,7 @@ require (
 	github.com/onsi/ginkgo/v2 v2.11.0
 	github.com/onsi/gomega v1.27.10
 	github.com/opencontainers/image-spec v1.1.0
-	oras.land/oras-go/v2 v2.4.0
```

Comment:
Can you also send another PR to update `.github/dependabot.yml` to cover `/test/e2e/`?

Reply:
That's a good idea. Will do.

Reply:
PR created: https://github.com/notaryproject/notation/pull/1030